modules/serverless-event-delivery-component-behaviors.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// serverless/event_workflows/serverless-event-delivery.adoc
+
+[id="serverless-event-delivery-component-behaviors_{context}"]
+= Event delivery behavior for Knative Eventing components
+
+Different Knative Eventing components have their own behavior patterns that are followed for event delivery.
+
+== Delivery failure status codes
+
+The channel or broker receiver may respond with the following status codes if an event fails to be delivered:
+
+`500`:: This is a generic status code which means that the event was not delivered successfully.
+`404`:: This status code means that the channel or broker the event is being delivered to does not exist, or that the `Host` header is incorrect.
+`400`:: This status code means that the event being sent to the receiver is invalid.
```

Comment:
wouldn't it be better to have a full list of the status codes that can occur (maybe even range), also the status codes that do indicate success (like 200, but there could be more. @openshift/team-knative-eventing do you have a comprehensive list of all HTTP status codes and ranges and what they mean in the context of Knative Eventing ?) As a developer, I would be happy if I would have a list of return codes collected at a single place without hunting those through the documentation.

Reply:
I added this section just for illustration purposes and don't want to expand the scope of this PR too much, which is intended just to show how to config event delivery. Can you maybe add an issue to the backlog to create an additional reference doc for this if you feel it's required?